src/TestWebsites/MVC/config/imageprocessor/cache.config

```diff
@@ -1,6 +1,6 @@
 ﻿<caching currentCache="DiskCache">
   <caches>
-    <cache name="DiskCache" type="ImageProcessor.Web.Caching.DiskCache, ImageProcessor.Web" maxDays="365">
+    <cache name="DiskCache" type="ImageProcessor.Web.Caching.DiskCache, ImageProcessor.Web" maxDays="365" browserMaxDays="365">
```

Comment:
This will need adding to the embedded config file also in the .Web project. I can do that though if you like?